components/AuthenticatedSidebar.tsx

```diff
@@ -55,8 +52,6 @@ function AuthenticatedLayout(props: any) {
       <span
         className={styles.item}
         onClick={async () => {
-          const token = Cookies.get(C.auth);
-          const response = await R.del(`/user/api-keys/${token}`, props.api);
```

Comment:
I think this is correct. When you sign in again you will create another token, so this guarantees the current token gets cleaned up. But happy to be persuaded the other way.